CONTRIBUTING.md

```diff
@@ -104,21 +104,17 @@ https://stestr.readthedocs.io/en/stable/MANUAL.html#test-selection
 If you want to run a single test module, test class, or individual test method you can
 do this faster with the `-n`/`--no-discover` option. For example, to run a module:
 ```
-tox -- -n test.python.test_examples
+tox -epy310 -- -n test.framework.test_composite
 ```
-Or to run the same module by path:
 
-```
-tox -- -n test/python/test_examples.py
```

Comment:
I wish this worked :slightly_smiling_face: It would make tab completion easier. I try it sometimes because it's what pytest uses and I forget it's not what unittest uses.